cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -105,6 +113,31 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        resources:
+          requests:
+            cpu: 150m
+            memory: 10Mi
+      - name: dnsmasq-metrics
+        image: gcr.io/google_containers/dnsmasq-metrics-amd64:1.0
+        livenessProbe:
+          httpGet:
+            path: /metrics
+            port: 10054
+            scheme: HTTP
+          initialDelaySeconds: 60
+          timeoutSeconds: 5
+          successThreshold: 1
+          failureThreshold: 5
+        args:
+        - --v=2
+        - --logtostderr
+        ports:
+        - containerPort: 10054
+          name: metrics
+          protocol: TCP
+        resources:
+          requests:
+            memory: 10Mi
```

Comment:
please add a comment about how this is computed, since it looks surprisingly low for a cache. I'm looking for something like each entry takes X and default number of entries is Y so we don't need more than XY.